src/context/Filters.context.tsx

```diff
@@ -0,0 +1,78 @@
+import { createContext, useState } from 'react';
+
+import type { Category } from '@/interfaces/Category';
+import { SortByEnum } from '@/interfaces/Filters';
+
+interface FiltersContextInterface {
+    search: string;
+    sortBy: SortByEnum;
+    category: '' | Category['id'];
```

Comment:
why number or empty string?
if it is because of initial value, initialise value with `null`

Reply:
I directly passing category to fetch. If I set to null, I'll need to add condition (`category ? category : ''`) before fetching data